app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -149,7 +148,7 @@ class NavigationOverlayFragment : Fragment() {
     private lateinit var pocketViewModel: PocketViewModel
     private lateinit var hintViewModel: HintViewModel
 
-    private lateinit var tileAdapter: PinnedTileAdapter
+    private var tileAdapter: PinnedTileAdapter? = null
```

Comment:
Why did you decide to make this nullable?

Reply:
I wanted to move the subscription into `onStart`, since that's where we're setting up all of our existing subscriptions.  But the adapter winds up being used before that gets called, so I needed null checks.  The alternative was to add a new `CompositeDisposable`, which seemed like too much complexity.  I tested this pretty thoroughly and didn't encounter any problems.